packages/itwin/tree-widget/package.json

```diff
@@ -112,5 +112,16 @@
     "@itwin/presentation-components": "^4.0.0-dev.30",
     "react": "^17.0.2",
     "react-dom": "^17.0.2"
+  },
+  "eslintConfig": {
+    "plugins": ["@itwin"],
+    "extends": ["plugin:@itwin/ui"],
+    "rules": {
+      "@itwin/no-internal": ["error"],
+      "@typescript-eslint/no-floating-promises": [
+        "error",
+        { "ignoreIIFE": true }
```

Comment:
are we using iife?

Reply:
Yes, this disables eslint error for this https://github.com/iTwin/viewer-components-react/blob/5ec78ebda7e1ec9da750439efa0de9437e2c28c5/packages/itwin/tree-widget/src/components/trees/category-tree/CategoriesTreeComponent.tsx#L50

Reply:
gross lol